Sources/AsyncDNSResolver/dnssd/DNSResolver_dnssd.swift

```diff
@@ -179,6 +176,27 @@ struct DNSSD {
 
 @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
 extension DNSSD {
+    // Class used to manage both the handler and the serviceRef pointers, so that they are deallocated when we are done using them
+    class Query {
+        let serviceRefPtr = UnsafeMutablePointer<DNSServiceRef?>.allocate(capacity: 1)
```

Comment:
```suggestion
        let serviceRefPointer = UnsafeMutablePointer<DNSServiceRef?>.allocate(capacity: 1)
```

Sorry I know "ptr" is from the original code, but let's change it to "pointer" to be consistent with `replyHandlerPointer`?